python-sdk/nuscenes/nuscenes.py

```diff
@@ -717,7 +719,7 @@ def crop_image(image: np.array,
         # Rotate image.
         ypr_rad = Quaternion(pose['rotation']).yaw_pitch_roll
         yaw_deg = -math.degrees(ypr_rad[0])
-        if rotate_lidar:
+        if ego_to_lidar:
             # Note that this is an approximation. The rotation between ego and lidar frame is not exactly 90 degrees.
             yaw_deg += 90
```

Comment:
Unfortunately, this won't work generally but just for a specific hardware setup

Reply:
Okay. Let's remove the option altogether.